app/pages/notifications.spec.js

```diff
@@ -30,18 +32,20 @@ describe('Notifications', function() {
     });
 
     it('will notify when no notifications present', function() {
+      const stub = sinon.stub(Notifications.prototype, 'componentWillMount');
       wrapper = mount(<Notifications user={{ id: 1 }} />);
       assert(wrapper.contains(<span>You have no notifications.</span>));
+      stub.restore();
     });
   });
 
   describe('it correctly display projects', function() {
     beforeEach(function () {
       wrapper = shallow(
-        <Notifications user={{ id: 1 }} />,
```

Comment:
This sounds like to me that when the API calls is made needs to be refactored. Those API calls shouldn't be made if there is no user. L35 in notifications.jsx likely should change to `if (nextProps.user !== null && nextProps.user !== this.props.user) { this.getProjectNotifications(); }`  Forcing the instance to render without a user doesn't seem like it the right way to test the expected rendered output for when the user should be logged in to me.